redis/redis.go

```diff
@@ -73,6 +73,10 @@ func (store *RedisFeatureStore) Get(key string) (*ld.Feature, error) {
 		return nil, jsonErr
```

Comment:
The redis changes are still untested-- staging -> dogfood and integration harness -> staging are using the memory store